packages/vow/src/track-turns.js

```diff
@@ -4,9 +4,10 @@ import {
   getEnvironmentOption,
   environmentOptionsListHas,
 } from '@endo/env-options';
+import { annotateError, X } from '@endo/errors';
 
 // NOTE: We can't import these because they're not in scope before lockdown.
-// import { assert, details as X, Fail } from '@agoric/assert';
+// import { assert, X, Fail } from '@endo/errors';
```

Comment:
Uh oh. The refactor may be incorrect here. Or, more likely, this is a constraint for the track-turns.js in endo, but not for this copy in @agoric/vow

Reply:
Inserted comment to clarify later